### ISSUE TITLE
Consumer Search for Business Health Declaration

### ISSUE THREAD

**Issue by noweber** [jorgecotillo/green_light_health#21]
As a Consumer,
I want to to know if the business location I want to go to is safe or not
so that I can protect myself and my family by visiting a different business or location.
	
Acceptance Criteria:
- Consumers can find health declarations for a specific business they want to enter.
- Consumers can agree or disagree to these health declarations.
- The general definition of done for the product's user stories is met: https://github.com/jorgecotillo/green_light_health/wiki/PBI-Defintion-of-Done.

**Comment by nasadorian** [comment]
@noweber Is the user not expected to sign a single (per disease) agreement with us? Do we foresee a scenario where businesses want a specific health declaration of their own?

**Comment by noweber** [comment]
Yes. I expect that businesses will prefer to use their own health declaration. This item is not high priority, though. We likely won't get to it within the course.

**Comment by noweber** [comment]
@nasadorian  To clarify, I think the single agreement with us is a good starting point. However, if that agreement is not all encompassing or sufficient for some businesses then it will be a no-go for those businesses to adopt use the app.
